Mathlib/Topology/MetricSpace/ProperSpace/Real.lean

```diff
@@ -0,0 +1,65 @@
+/-
+Copyright (c) 2017 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl, Mario Carneiro
+-/
+import Mathlib.Data.Rat.Encodable
+import Mathlib.Topology.MetricSpace.Isometry
+import Mathlib.Topology.MetricSpace.ProperSpace
+import Mathlib.Topology.Order.Compact
+import Mathlib.Topology.Order.MonotoneContinuity
+import Mathlib.Topology.Order.Real
+import Mathlib.Topology.UniformSpace.Real
+
+/-!
+# Second countability of the reals
+-/
+
+assert_not_exists IsTopologicalRing UniformContinuousConstSMul UniformOnFun
+
+noncomputable section
+
+open Set Topology TopologicalSpace
+
+instance : ProperSpace ℝ where
+  isCompact_closedBall x r := by
+    rw [Real.closedBall_eq_Icc]
+    apply isCompact_Icc
+
+instance : SecondCountableTopology ℝ := secondCountable_of_proper
+
+namespace EReal
+
+instance : SecondCountableTopology EReal :=
+  have : SeparableSpace EReal := ⟨⟨_, countable_range _, denseRange_ratCast⟩⟩
+  .of_separableSpace_orderTopology _
+
+end EReal
+
+namespace NNReal
+
+/-!
+Instances for the following typeclasses are defined:
```

Comment:
Then you could condense this comment a bit, and just say "everything is inherited from the corresponding structures on the reals".